integration_tests/integration/court-register-amend.spec.ts

```diff
@@ -187,4 +188,33 @@ context('Court register - amend existing court', () => {
       CourtDetailsPage.verifyOnPage('Sheffield Magistrates Court').courtUpdatedConfirmationBlock().should('exist')
     })
   })
+  describe('amending a court building contacts', () => {
+    beforeEach(() => {
+      IndexPage.verifyOnPage().courtRegisterLink().click()
+      AllCourtsPagedPage.verifyOnPage()
+        .viewCourtLink('SHFMC')
+        .should('contain.text', 'Sheffield Magistrates Court')
+        .click()
+    })
+    it('should show summary of court with link to amend each set on contacts', () => {
+      const courtDetailsPage = CourtDetailsPage.verifyOnPage('Sheffield Magistrates Court')
+      courtDetailsPage.buildingDetailsSection('1').should('contain.text', 'Sheffield Courts')
+      courtDetailsPage.buildingDetailsSection('2').should('contain.text', 'Sheffield Court Annexe')
+      courtDetailsPage.amendBuildingContactsLink('1').should('contain.text', 'Change')
+      courtDetailsPage.amendBuildingContactsLink('2').should('contain.text', 'Change')
+    })
+    it('will display existing contact details with ability to remove a number', () => {
+      const courtDetailsPage = CourtDetailsPage.verifyOnPage('Sheffield Magistrates Court')
+      courtDetailsPage.amendBuildingContactsLink('1').click()
+      const amendCourtBuildingContactsPage = AmendCourtBuildingContactsPage.verifyOnPage('Sheffield Courts')
+
+      amendCourtBuildingContactsPage.number(0).should('have.value', '0114 555 1234')
+      amendCourtBuildingContactsPage.type(0).should('have.value', 'TEL')
+      amendCourtBuildingContactsPage.removeButton(0)
```

Comment:
Should we assert on the text in the remove button?

Reply:
I was going to say no, but actually since we get by id that would be good. Next PR